src/container/container.tsx

```diff
@@ -37,12 +37,26 @@ interface Props extends ContainerProps, Dispatcher, PropsFromState {
 
 interface ContainerState {
 }
+
+/**
+ * Retrieve the playerName for the given player.
+ * Will return the fixed configured name in `playerInfo.ts` first, or the
+ * name returned from the GSI events.
+ */
+function playerName(player: Player): string {
```

Comment:
Role of this file is just pass calculated value to React component.
So please move this function to https://github.com/Thiry1/csgo-custom-hud/blob/master/src/redux/modules/players/players.ts .
And apply function to https://github.com/Thiry1/csgo-custom-hud/blob/master/src/redux/modules/players/players.ts#L157 .

Reply:
Thanks for the explanation! Your English is very good and it very easy to understand no worries.

I hope it's not problem that I have to add comments in English as I don't know much more than 4 words of Japenese :/

Reply:
>I hope it's not a problem that I have to add comments in English
No problem!Using English is helpful for other people.
And I can study English :D